src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -27,6 +27,9 @@ public class MemoryCache : IMemoryCache
 
         private CoherentState _coherentState;
         private bool _disposed;
+        private readonly object _sync = new object();
+        private long _totalHits;
+        private long _totalRequests;
```

Comment:
A trick you can use it to store the number of hits & misses instead, saving one `Interlocked` operation on the hot path.
```c#
private long _totalHits;
private long _totalMisses;
```